include/stdexec/concepts.hpp

```diff
@@ -198,6 +198,10 @@ namespace stdexec {
       constructible_from<_T, _As...> && std::is_nothrow_constructible_v<_T, _As...>;
 #endif
 
+  template <class _Ty>
+    concept __decay_copyable =
+      constructible_from<decay_t<_Ty>, _Ty>;
+
   template <class _Ty>
     concept __nothrow_decay_copyable =
```

Comment:
For the sake of concept subsumption...

```suggestion
    concept __nothrow_decay_copyable =
      __decay_copyable<_Ty> &&
```

Reply:
I tested my change with something like 

```
template <class T> requires __nothrow_decay_copyable<T>
int fn(T t) { return 0; } 
template <class T> requires __decay_copyable<T>
int fn(T t) { return 1; }
```

And confirmed `T=int` (which satisfies both concepts) selected the first `fn`. Perhaps my test was not completely valid to verify subsumption works?

Reply:
Ah, sure. Because `__nothrow_constructible_from` subsumes `constructible_from`. Disregard.